### PR TITLE
Finishing the navigation bars

### PR DESCRIPTION
**Pull request by BanTheNons**
I haven't finished it yet so please don't merge

### CODE REVIEW

File: components/Navigation.tsx [hunk 1]
```diff
@@ -645,21 +645,267 @@ export default function Navigation() {
       </NavigationSection>
 
       <NavigationSection title="Topics">
-        <NavigationLink href="/topics/certified-devices">
+        <NavigationLink
+          href="/topics/certified-devices"
+          subLinks={
+            <Fragment>
+              <NavigationSubLink href="/topics/certified-devices#hows-it-work">
```
Comment (advaith1) [inline components/Navigation.tsx]:
```suggestion
              <NavigationSubLink href="/topics/certified-devices#how's-it-work?">
```

File: components/Navigation.tsx [hunk 2]
```diff
@@ -645,21 +645,267 @@ export default function Navigation() {
...
+              <NavigationSubLink href="/topics/certified-devices#connecting">
+                Connecting
+              </NavigationSubLink>
+              <NavigationSubLink href="/topics/certified-devices#getting-device-uuid">
+                Getting Device UUID
+              </NavigationSubLink>
+              <NavigationSubLink href="/topics/certified-devices#http-example">
+                HTTP Example
+              </NavigationSubLink>
+              <NavigationSubLink href="/topics/certified-devices#websocker-example">
```
Comment (advaith1) [inline components/Navigation.tsx]:
```suggestion
              <NavigationSubLink href="/topics/certified-devices#websocket-example">
```

File: components/Navigation.tsx [hunk 3]
```diff
@@ -645,21 +645,267 @@ export default function Navigation() {
...
+              </NavigationSubLink>
+              <NavigationSubLink href="/topics/community-resources#intent-calculators">
+                Intent Calculators
+              </NavigationSubLink>
+              <NavigationSubLink href="/topics/community-resources#api-types">
+                API Types
+              </NavigationSubLink>
+            </Fragment>
+          }
+        >
```
Comment (advaith1) [inline components/Navigation.tsx]: is this intentionally missing Embed Visualizer?

File: components/Navigation.tsx (hunk 1 above)
Comment (BanTheNons) [reply components/Navigation.tsx]: I didn't realize this before but you actually need to include special characters for the url to actually work.

**Comment by BanTheNons** [comment]
Time for the game sdk, oh boy

**Pull request merged**